provider/virt_storage/backend/ceph.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from provider.virt_storage.helper import rbdcli
+
+from provider.virt_storage import storage_volume
+from provider.virt_storage.backend import base
+
+
+class CephPool(base.BaseStoragePool):
+    TYPE = "rbd"
+
+    def __init__(self, name):
+        self.image = None
+        self.server = None
+        super(CephPool, self).__init__(name)
+
```

Comment:
We may have to overwrite the pool_define_by_params function to define the target and source, not sure why target is included in the pool, refer to libvirt target should be in the volume, anyway, let's keep consistent with the current design.

@classmethod
    def pool_define_by_params(cls, name, params):
        inst = cls(name)
        inst.target = virt_target.PoolTarget.target_define_by_params(params)
        inst.target.path = params['rbd_pool_name']
        source_params = params.object_params(name)
        inst.source = virt_source.PoolSource.source_define_by_params(
                name, source_params)
        inst.set_special_opts_by_params(params)
        return inst

Reply:
Have updated, thanks zhenchao.